webapp/components/post_view/components/post_list.jsx

```diff
@@ -392,15 +392,15 @@ export default class PostList extends React.Component {
         if (this.props.scrollType === ScrollTypes.BOTTOM) {
             this.scrollToBottom();
         } else if (this.props.scrollType === ScrollTypes.NEW_MESSAGE) {
-            window.setTimeout(window.requestAnimationFrame(() => {
+            window.requestAnimationFrame(() => {
```

Comment:
We were already using it here, I just remove the `setTimeout` so it should be fine I think